src/main.rs

```diff
@@ -1,8 +1,8 @@
-use clap::{Arg, App, SubCommand};
+use clap::{App, Arg, SubCommand};
 
 use std::path::Path;
 
-const PATH: &str = "/Users/carllerche/Code/Tokio/tokio";
+const PATH: &str = "/usr/local/var/foss/tokio";
```

Comment:
What about updating the CLI to take a `--project-path` flag to specify the path instead of hard coding one? 👍

Reply:
would you prefer to prompt the user to input the path by passing `--project-path`? I missed this detail, sorry